lib/app/views/application/_nav.erb

```diff
@@ -2,6 +2,22 @@
   <ul class="nav navbar-nav">
     <li><a href="http://help.exercism.io">Help</a></li>
     <li><a href="/getting-started">Welcome new users</a></li>
+    <li class="dropdown">
+      <a class="dropdown-toggle" id="drop-current" href="#" data-toggle="dropdown">Languages<b class="caret"></b></a>
+    <ul class="dropdown-menu" id="menu-current" role="menu" area-labelledby="drop-current">
+          <% all_languages.each do |l| %>
```

Comment:
A completely separate thing: The current codebase is a right mess, and there is plenty of terrible code... and I apologize! In particular, I bet you just emulated some style from other code in the codebase here, and so my complaints (coming right up :) ) might sound a bit disingenuous... I think that there's some interesting stuff to learn, though, so let's talk about these things anyway!

We already talked about `languages` vs `tracks`, so I'm not going to go into that. Let's just pretend that languages was the correct name for this.

Look at the following three examples, and think about which you like best, and also make up an opinion about why you like one or the other best (it doesn't have to be "right" or anything, I just think it's worth making up an argument which can then form the basis for a discussion).

```ruby
all_monkeys.each do |m|
  m.see and m.do
end
```

```ruby
tribe.each do |monkey|
  monkey.see and monkey.do
end
```

```ruby
monkeys.each do |monkey|
  monkey.see and monkey.do
end
```